### PR TITLE
Some addition on the node pools migration

### PR DESCRIPTION
**Pull request by ppatierno**
### Type of change

- Documentation

### Description

This PR adds a couple of more information to make the user aware of what's happening during the node pools migration.

### CODE REVIEW

**Review by PaulRMellor — COMMENTED** [review]
Couple of suggestions. 
I don't have any strong opinions on whetehr we need to include the full message or not. I think it's easier to read without it

**Review by scholzj — COMMENTED** [review]
TBH, I'm not sure I follow the value of this. the warning is quite self-explanatory. If you don't think so, you should likely fix the watning and not explain it in the docs.

**Comment by ppatierno** [comment]
I think the documentation should help to understand what to expect during that step and explain what's happening. I think it's useful tbh.

**Comment by scholzj** [comment]
Well, I'm not going to block it. But I think it is risky to copy log messages to the docs as they might change easily. But also, you are basically saying _the message you see in the log means exactly what it says it means_.

**Comment by scholzj** [comment]
Just to be clear, I do not want to block it, I just shared my view. If you and Paul are fine with it feel free to proceed with it.

**Comment by ppatierno** [comment]
I still think that by reading the doc while running the steps it's useful to see what's going to happen.
Maybe reporting the log message is not the best way. I will use Paul's suggestion.

File: documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc [hunk 1]
```diff
@@ -67,6 +67,10 @@ env:
   - name: STRIMZI_FEATURE_GATES
     value: +KafkaNodePools
 ----
++
+After restarting, the Cluster Operator will log the following warning message:
+"KafkaNodePool kafka in namespace _<namespace>_ was ADDED, but the Kafka cluster my-cluster to which it belongs does not have KafkaNodePool support enabled".
+It means that, while the node pool for the Kafka brokers was created, it is not handled by the Cluster Operator yet.
```
[the remark this answers is not in the document]
Comment (ppatierno) [reply documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc]: @PaulRMellor I went with the second suggestion.

**Comment by PaulRMellor** [comment]
I think this update is fine. It acts as a verification that the procedure is going in the right direction.

**Review by PaulRMellor — APPROVED** [review]

**Pull request merged**